ironfish/src/rpc/adapters/ipcAdapter.test.ts

```diff
@@ -78,7 +78,7 @@ describe('IpcAdapter', () => {
     await ipc.start()
     await client.connect()
 
-    const response = client.request('foo/bar')
+    const response = client.request('foo/bar', 'test token')
```

Comment:
Can we inject this into the client and have that auto insert this into each request instead of having to pass it into each request here?